README.md

```diff
@@ -47,6 +48,11 @@ This project is inspired by the original design idea by Gregoire Vella on Behanc
 ![Alt text](image-1.png)
 ![Alt text](image-2.png)
 ![Alt text](image-3.png)
+
+### Acknowledgment <a name="Acknowledgment"></a>
+
+We would like to express our deepest gratitude to **Gregoire Vella**, the talented app designer who contributed significantly to the visual and user experience aspects of this project. His creative insights and dedication played a crucial role in shaping the design of our application. We are truly thankful for his expertise and valuable contributions.
+
```

Comment:
- [ ] Firstly, kindly add the associated links to the name of the design owner. This way, potential people of interest in your work will have a path to navigate, to confirm the authenticity of the owner 👍  

- [ ] Secondly, the `Acknowledgement` section of the README is positioned at the bottom of the README setup, as this particular detail is meant to be at the concluding part of your work to give credit.
- Please adjust the position as in the [Template](https://github.com/microverseinc/readme-template) 👍 

### Template:
![image](https://github.com/PabloBona/budget/assets/50990925/6b78aa57-cd57-4819-b455-dd229f68188e)

### Design owner needed links:
![image](https://github.com/PabloBona/budget/assets/50990925/8fe93b07-3e7a-440f-aab6-b838e557c8d5)